### PR TITLE
Added a Klipper configuration and wiring diagram for the V1.8 using a…

### PR DESCRIPTION
**Pull request by mhoush**
… FYSETC Spider 1.1 board

  * [ ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ ] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ ] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ ] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
This adds an example configuration and wiring diagram for a FYSETC Spider 1.1 board on a V1.8 printer.

#### How was it tested? 
So far I've been using it on my own printer without issues, but my sample size is 1. Feedback is welcome.

#### Further notes
Greetings! Here's an initial Klipper configuration and wiring diagram for the FYSETC Spider and a V1. I used the existing SKR 1.4 configuration and wiring diagrams as bases for style and format. The pin definitions came from the upstream git repository and links to said git repository are also in the config file. Included is a wiring diagram based on how I have it connected to my V1 currently.

Hopefully the style/format is good since it's based on the SKR 1.4 but of course if I missed anything or messed it up somewhere let me know and I'll fix it. Hopefully this can be a good starting point to helping others with their Spider V1 builds! Questions/comments/feedback welcome. jaeger#1794 on discord.

### CODE REVIEW

File: firmware_configurations/klipper/jaeger/Spider_1.1_V1.8/readme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<B>VORON V1.8 FYSETC Spider Config</B>
+
```
Comment (KiloQubit) [inline firmware_configurations/klipper/jaeger/Spider_1.1_V1.8/readme.md:2]: Can you add your image to this README?  Something like this

    Spider Wiring

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Just a minor change in the README below.

File: firmware_configurations/klipper/jaeger/Spider_1.1_V1.8/readme.md (hunk 1 above)
Comment (mhoush) [reply firmware_configurations/klipper/jaeger/Spider_1.1_V1.8/readme.md:2]: Sure, will do. Thanks for the review!

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!

**Comment by mhoush** [comment]
No problem!

**Pull request merged**